presto-main/src/main/java/io/prestosql/sql/planner/planPrinter/NodeRepresentation.java

```diff
@@ -137,4 +137,26 @@ public String getDetails()
     {
         return estimatedCost;
     }
+
+    public static class OutputSymbol
```

Comment:
`TypedSymbol`, `SymbolAndType`?  "Output" is a property of how it's used, but the concept this represents is a symbol + its type.

Reply:
Addressed as separate commit, I do not feel comfortable editing (commit) code that is authored by some else. Should I squash it?

Reply:
Yes, please squash it. That is ok, it’s similar to applying comments for someone else’s change before merging a PR.